assets/css/style.css

```diff
@@ -0,0 +1,127 @@
+body {
+    background-image: url("../images/bananas.png")
+}
+
+.top-page {
+    position: relative;
```

Comment:
you do not need position relative to move right and left, we can use only a few things, so if you remove this line and the others noted and then add the css requested you'll see similar results with less css.  This css also gets applied to the children so it can make it hard to style with some of these properties.